src/writeData/subscriptions/components/BrokerDetails.tsx

```diff
@@ -56,11 +57,18 @@ const BrokerDetails: FC<Props> = ({
 }) => {
   const history = useHistory()
   const org = useSelector(getOrg)
+  const {navbarMode} = useContext(AppSettingContext)
+  const navbarOpen = navbarMode === 'expanded'
   return (
     <div className="update-broker-form" id="broker">
       <SpinnerContainer spinnerComponent={<TechnoSpinner />} loading={loading}>
         <Form onSubmit={() => {}} testID="update-broker-form-overlay">
-          <div className="update-broker-form__fixed">
+          <div
+            className="update-broker-form__fixed"
+            style={{
+              width: navbarOpen ? '61%' : '69%',
```

Comment:
I was wondering why don't we make the body portion a flexbox so that the width is automatically handled?

Reply:
I've tried a few different ways but with the position fixed this was the only way I could get it to line up properly to the form. Open to suggestions though if you wanna jump on a zoom!

Reply:
As discussed in zoom the page would need a refactor to implement flex here so we are parking it for now as this is fixing the immediate bugs :)